src/DynamoCore/Core/PreferenceSettings.cs

```diff
@@ -59,7 +59,7 @@ public class PreferenceSettings : NotificationObject, IPreferences
         /// <summary>
         /// Should the background 3D preview be shown?
         /// </summary>
-        public bool FullscreenWatchShowing { get; set; }
+        public bool IsBackgroundPreviewActive { get; set; }
```

Comment:
It will not have an issue, it will just fail to find the property and will set it to the default, `true`. I did a bit of studying on this. You can't deserialize to a property whose name is changed. I would have thought there was an attribute to help with this. But there is not. The only option is to create a new property, allow order files to deserialize to the old property and then synch the value with the new property.